config/helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
 version: 0.9.1
-appVersion: 1.6.0
+appVersion: 1.6.1
```

Comment:
the app version should stay 1.6.0, the chart version should be incremented when only modifying the helm chart.

Reply:
Heh yep. I must've absentmindedly incremented the wrong one.